include/macroni/Common/EmptyVisitor.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "vast/CodeGen/CodeGenBuilder.hpp"
+#include "vast/CodeGen/CodeGenMeta.hpp"
+#include "vast/CodeGen/CodeGenVisitorBase.hpp"
+#include "vast/CodeGen/SymbolGenerator.hpp"
+#include "vast/Util/Common.hpp"
+
+namespace macroni {
+struct empty_visitor : vast::cg::visitor_base {
```

Comment:
there is `vast::cg::::fallthrough_visitor` that behaves the same

Reply:
Is this in the latest version of vast? I am still using the mx_integration branch (commit a364ea3b3bae7e77909cb59655172c986570bf50)

Reply:
It is already merged in upstream VAST then.